documentation/airport-guides/ltba.md

```diff
@@ -63,9 +63,22 @@ The sim features a combination of these sectors:
 
 ### Other aerodromes of interest
 
+##### Istanbul New Airport, LTFM
+*Istanbul New Airport*  is an international airport under construction in the *Arnavutköy* district on the European side of Istanbul.
+The reason for the construction is the lack of capacity of *Istanbul Atatürk Airport*, which has reached its capacity limits.
```

Comment:
Add this at the end of the previous sentence and delete this one
`...on the European side of Istanbul, with the aim of replacing the current *Istanbul Atatürk Airport*, which has reached its capacity limits.`